packages/postcss-plugins-preset/lib/index.js

```diff
@@ -4,7 +4,7 @@
 const { adminColorSchemes } = require( '@wordpress/base-styles' );
 
 module.exports = [
+	require( 'postcss-custom-properties' )(),
 	require( '@wordpress/postcss-themes' )( adminColorSchemes ),
```

Comment:
If we switch these two presets priorities,  the admin schemes won't work on IE11 (the default one still works) but we'll gain in terms of size of the production CSS build. I think it's fine to do it but I'll bring this to #core-css meetings.

Reply:
The postcss-themes plugin also causes issue with over-specificity of color scheme rules, that's why the inspector panel headings are colored. The generated rule `body.admin-color-ectoplasm .components-button[aria-expanded="true"]` is overriding the "correct" `.components-panel__body-toggle.components-button` rule, which is set to plain black, not a color scheme color.

If these are swapped, only the `:root` is generated for each scheme, which fixes that specificity issue.

IMO it's okay to only have default on IE11, since there are these other drawbacks to keeping support.

Reply:
I updated the setup, it's way better now. IE11 only shows the default colors.